conf/defaults.ini

```diff
@@ -1444,6 +1444,9 @@ public_key_retrieval_disabled = false
 # Force download of the public key for verifying plugin signature on startup. If disabled, the public key will be retrieved every 10 days.
 # Requires public_key_retrieval_disabled to be false to have any effect.
 public_key_retrieval_on_startup = false
+# Enter a comma-separated list of plugin identifiers to load external versions of core plugins. The external version of the plugin needs to be explicitly installed.
+# Experimental. Requires the feature toggle decoupleCorePlugins to be enabled. 
+load_external_plugins =
```

Comment:
Nit. Naming is hard. Feels like something missing because you can't get a distinct understanding of it by just reading the name of the setting Would any of these this make more sense/read easier?

```
load_core_as_external_plugins = 

[plugin.<plugin id>]
use_external = true
```

Reply:
Okay :+1: